config/initializers/session_store.rb

```diff
@@ -0,0 +1,18 @@
+redis_url = ENV.fetch("REDIS_URL", nil)
+
+if redis_url.nil? || Rails.env.test?
+  Rails.application.config.session_store :cookie_store, key: "_roda_session"
+else
+  redis_uri = URI(redis_url)
+  Rails.application.config.session_store :redis_store,
+    servers: {
+      host: redis_uri.host,
+      port: redis_uri.port,
+      db: 1,
+      namespace: "roda:session",
+    },
+    expire_after: 12.hours,
+    threadsafe: true,
+    signed: true,
+    secure: true
```

Comment:
Do we also need `key` here?

Reply:
From testing `namespace: "roda:session"` has an affect on the key name but the `key` argument doesn't. This might be because we have the `redis-namespace` gem in the mix too.

In this screenshot of my local session keys you can see the first few entries where only `key` is used and the last where the `namespace` is used.

![Screenshot 2020-03-18 at 13 00 30](https://user-images.githubusercontent.com/912473/76963143-76b61f00-6918-11ea-8644-610f9b86fac2.png)

I figure that we don't but I don't really like the magic going on as it does say `key` in `redis-rails` 🤷‍♂

Reply:
Odd, my take on `key` is it the name of the cookie used locally - mine blows up without it!?

Reply:
here is mine set to `_roda_session` in FF:

![Screenshot 2020-03-18 at 13 08 49](https://user-images.githubusercontent.com/480578/76963874-a31e6b00-6919-11ea-82bf-d53997f481be.png)

Reply:
Doh 🤦‍♂ yes of course there is a redis key **and** a client key. Updated!